Assets/Mirror/Runtime/Transport/Websocket/Client.cs

```diff
@@ -121,15 +126,15 @@ async Task<ArraySegment<byte>> ReadFrames(WebSocketReceiveResult result, WebSock
 
             while (!result.EndOfMessage)
             {
-                if (count >= MaxMessageSize)
+                if (count >= _maxMessageSize)
```

Comment:
```suggestion
                if (count > _maxMessageSize)
```

it is ok if count == _maxMessageSize,  we have enough space for the entire message